doc/ref_model/chapters/chapter04.md

```diff
@@ -420,23 +420,25 @@ n100, n200, n300, n400, n500, n600 |100, 200, 300, 400, 500, 600 Gbps
 <a name="4.2.3"></a>
 ###  4.2.3 Storage Extensions
 
-Multiple persistent storage extensions can be attached to virtual compute instances for data storage. Each extension can be configured with the required performance category.
+Persistent storage is associated with VNFs via Storage Extensions. The size of an extension can be specified explicitly in increments of 100GB, ranging from a minimum of 100GB to a maximum of 16TB. Extensions are configured with the required performance category, as per Table 4-19. Multiple persistent Storage Extensions can be attached to virtual compute instances.
 
-.conf |Read IO/s |Write IO/s Read |Throughput (MB/s) |Write Throughput (MB/s)
----|---|---|---|---
-.bronze |Up to 3K |Up to 15K |Up to 180 |Up to 120
-.silver |Up to 60K |Up to 30K |Up to 1200 |Up to 400
-.gold |Up to 680K |Up to 360K |Up to 2650 |Up to 1400
+Note, CNTT documentation uses GB and GiB to refer to a Gibibyte (2<sup>30</sup> bytes), except where explicitly stated otherwise.
+
+.conf |Read IO/s |Write IO/s Read |Throughput (MB/s) |Write Throughput (MB/s) |Max Ext Size
+---|---|---|---|---|---
+.bronze |Up to 3K |Up to 15K |Up to 180 |Up to 120 |16TB
+.silver |Up to 60K |Up to 30K |Up to 1200 |Up to 400 |1TB
+.gold |Up to 680K |Up to 360K |Up to 2650 |Up to 1400 |300GB
```

Comment:
I think it would be good to articulate the rationale for the maximum sizes.  The way I'd been thinking of it was more along the lines of:

|.conf|IOPS per GB|Throughput per GB|
|.bronze|low|low|
|.silver|medium|
|.gold|high|

My thinking being that a NF vendor would choose an instance profile, and then select a storage extension (gold/silver/bronze) and then a size - what I'm not seeing here at the moment is what drives a maximum value. i.e. how would it affect the certification of a VNF/CNF as part of CNTT/OVP?

Reply:
@tomkivlin Hi Tom,
>
> ...a NF vendor would choose an instance profile, and then select a storage extension (gold/silver/bronze) and then a size

I'm thinking they should be able to do that as currently written. Do you see something that would inhibit that process?

> ...what I'm not seeing here at the moment is what drives a maximum value. i.e. how would it affect the certification...

I concur, the drivers aren't listed. However, that's pretty consistent w/ how we've written most of the doc. For example, we don't provide a narrative on how the limits for RAM, number of cores, etc. were arrived at, either. Do you think this is a corner case?

Reply:
@markshostak let's park the narrative part - as you say this is consistent so let's take that outside this PR.

> I'm thinking they should be able to do that as currently written. Do you see something that would inhibit that process?

Only that, unless I'm mistaken, the performance of a storage extension will be driven by the size of it.  i.e. you're not going to get 2650Mbps of storage throughput from a 10G gold extension.  So, what I don't feel I'm able to do at the moment is, as someone who will be designing a workload, or deploying a workload, is to say "I need x throughput, therefore I need a gold storage extension yGB in size, even if the workload doesn't actually need that much GB capacity."

Reply:
@tomkivlin I'd say the required performance impacts the maximum extension size, but that's intentional. Specifically, the rationale is related to how much space NVMe servers have these days in practice. Also, worse case, if you need more, you can attach multiple extensions to achieve the space you need, assuming it's available and you don't get shot for hogging a precious resource.